### PR TITLE
chore: bump dependencies

### PR DESCRIPTION
**Pull request by tserg**
This PR bumps Cairo and related dependencies to `v2.6.3`.

### CODE REVIEW

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -13,11 +13,14 @@ jobs:
       - uses: actions/checkout@v3
       - uses: software-mansion/setup-scarb@v1
         with:
-          scarb-version: "2.4.0"
+          scarb-version: "2.6.3"
       - run: scarb fmt --check
       - run: scarb build
 
+      - name: Install Universal-Sierra-Compiler
```
Comment (milancermak) [inline .github/workflows/tests.yml:20]: I thought this is installed as a dependency of SNFoundry, but it seems it's not the case in the CI. I think we have to bump `setup-snfoundry` to [v3](https://github.com/foundry-rs/setup-snfoundry/releases/tag/v3.0.0) to get it. Can you try?

File: Scarb.toml [hunk 2]
```diff
@@ -10,10 +10,10 @@ license-file = "LICENSE"
 keywords = ["fixed-point", "wad", "ray", "cairo", "starknet"]
 
 [dependencies]
-starknet = ">= 2.4"
+starknet = ">= 2.6.3"
```
Comment (milancermak) [inline Scarb.toml:13]: I'd be a little less strict here and allow >= 2.6.0.

**Review by milancermak — APPROVED** [review]
LGTM, left just small stuff comments.

File: .github/workflows/tests.yml (hunk 1 above)
Comment (tserg) [reply .github/workflows/tests.yml:20]: oh this sounds likely, I was looking for the wrong thing - a CI for universal-sierra-compiler. I will give this a try.

**Pull request merged**